.docker/src-cms/etc/periodic/1min/call-center-update

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
```

Comment:
I am still getting this message: 
```
cms       | run-parts: can't execute '/etc/periodic/1min/call-center-update': No such file or directory
```
The problem is /bin/bash -- we don't have that on the cms. I changed it to /bin/ash and got this instead: 

```
cms       | S3 service storage does not exist in space 
cms       | run-parts: /etc/periodic/1min/call-center-update: exit status 1
```

A lot of this script looks like it's supposed to run in a developer's terminal. The good news is that we already have code in /scripts/bootstrap to get the S3 environment variables (from $VCAP_SERVICES). Bootstrap would need to insert those into the call-center-update script or otherwise make them available.